benchexec/containerexecutor.py

```diff
@@ -647,6 +647,9 @@ def grandchild():
                 # Signal readiness to parent by sending our PID
                 # and wait until parent is also ready
                 os.write(to_parent, str(my_outer_pid).encode())
+                logging.debug(
```

Comment:
While these log statements may be useful during development of this PR, I don't think they are that useful further on, and especially I think they clutter the output too much (something like `runexec --debug` is also commonly used to debug problems with the system setup, so it is good if there is not too much of the purely internal and developer-only debug messages).

I would suggest to remove all of the added log messages.

Reply:
Removed